src/graph.rs

```diff
@@ -470,7 +471,68 @@ fn is_false(v: &bool) -> bool {
   !v
 }
 
-#[derive(Debug, Default, Clone, Serialize, PartialEq, Eq)]
+fn serialize_import_assertions<S>(
+  assertions: &ImportAssertions,
+  serializer: S,
+) -> Result<S::Ok, S::Error>
+where
+  S: Serializer,
+{
+  if let Some(map) = assertions.maybe_static() {
+    map.serialize(serializer)
```

Comment:
This doesn't seem exactly right because we're losing resolution. Overall, I think we should maybe remove the serialization for this completely atm since import assertions might change a lot https://github.com/denoland/deno/issues/17944

Reply:
Done. And since errors are unstructured strings, this doesn't stop us from still including assertion errors in `Import::errors`.